styles.css

```diff
@@ -631,14 +646,108 @@ h6 {
 }
 
 .form button {
-  display: flex;
+  /* display: flex; */
   justify-content: center;
   align-items: center;
   text-align: center;
-  color: #6070ff;
+
+  /* color: #6070ff; */
 }
 
 .get-in-touch {
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (min-width: 768px) {
+  .Template-1 {
+    position: absolute;
+    width: 100%;
+  }
+
+  .header-background-mobile {
+    background-image: url(./media/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .Toolbar img {
+    display: none;
+  }
+
+  .Toolbar ul {
+    display: flex;
+    list-style-type: none;
+    color: #344563;
+  }
+
+  .Card-Works {
+    flex-direction: row;
+    height: 496px;
+  }
+
+  .Snapshoot-Portfolio img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .Left-Block {
+    width: 544px;
+    height: 445px;
+  }
+
+  .About-myself {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
```

Comment:
- [ ] Good job on your design 👍🏽 . Kindly ensure that you adjust the spacing of your about-section to match the Figma design. Please adjust the margins/paddings😄

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/199581942-470d2f0d-8162-4e17-bbd4-d490b5d391d3.png)  | ![image](https://user-images.githubusercontent.com/63473584/199582185-4e7f41d1-0050-493b-861f-26111526d6c6.png)  |